uint/src/uint.rs

```diff
@@ -1577,7 +1580,8 @@ macro_rules! impl_quickcheck_arbitrary_for_uint {
 			fn arbitrary<G: $crate::quickcheck::Gen>(g: &mut G) -> Self {
 				let mut res = [0u8; $n_bytes];
 
-				let p = g.next_f64();
+				use $crate::rand::Rng;
```

Comment:
Can you explain this change?

It seems unnecessary and requires to have direct dendepency to `rand`, see https://docs.rs/quickcheck/0.9.0/quickcheck/trait.Gen.html.